conftest.py

```diff
@@ -104,10 +104,20 @@ def _loaded_contract_fixtures(populus_source_root, project_dir, request):
             'fixtures',
             item,
         )
+        
+        fixture_path_remap = os.path.join(
+            populus_source_root,
+            'tests',
+            'fixtures_remap',
+            item,
+        )
+        
```

Comment:
we don't need a second path for fixtures in the `tests/` directory of the project.  The `./tests/fixtures` directory is suitable for all source file fixtures.  When we access fixtures in tests they are *always* copied to be sure that nothing is being mutated or modified during test runs.